Update kernel helm chart

tooxie opened the pull request:
Introduce the improvements that we already have for the UI chart.

tooxie committed:
Update kernel helm chart

Introduce the improvements that we already have for the UI chart.

MrPink reviewed (dismissed):
LGTM

shawnsarwar reviewed (commented):
I need to get up to speed on our kube facing processes before I can competently review this. I defer to Fred and our Will.

tooxie committed:
chore: Make pods fail if Postgres is not ready

tooxie committed:
fix: create an odk secret resource

tooxie committed:
fix: kernel token must match

tooxie committed:
fix: add variables for odk

tooxie committed:
fix: duplicate nginx config for the 2 servers

We have a `RELEASE-NAME.dev.ehealthafrica.org` server and a
`RELEASE-NAME` server, the only difference is that the former needs an
https redirect while the latter breaks odk tests if it has it.

quesebifurcan commented on helm/kernel/templates/NOTES.txt:
```diff
@@ -1,19 +1 @@
-  echo "Visit http://127.0.0.1:8080 to use your application"
-  kubectl port-forward $POD_NAME 8080:{{ .Values.service.internalPort }}
-{{- end }}
+Visit http://{{ .Release.Name }}.{{ .Values.domain }} to use your application
```
Phew, much better.

tooxie replied:
Even though now it's not accurate for minikube deployments 😅 but since that's only done in travis, I decided to ignore it.
